cvat-ui/src/reducers/jobs-reducer.ts

```diff
@@ -89,6 +92,54 @@ export default (state: JobsState = defaultState, action: JobsActions): JobsState
                 },
             };
         }
+        case JobsActionTypes.DELETE_JOB: {
+            const { jobID } = action.payload;
+            const { deletes } = state.activities;
+
+            deletes[jobID] = false;
+
+            return {
+                ...state,
+                activities: {
+                    ...state.activities,
+                    deletes: {
+                        ...deletes,
+                    },
+                },
+            };
```

Comment:
Avoid making state object mutable

```suggestion
            return {
                ...state,
                activities: {
                    ...state.activities,
                    deletes: {
                        ...deletes,
                        [jobID]: false,
                    },
                },
            };
```